### PR TITLE
Add mapping for atlas webapp

### PR DESCRIPTION
**Pull request by Vampouille**

### CODE REVIEW

File: atlas/atlas.properties [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+psql.url=jdbc:postgresql://localhost:5432/georchestra
+psql.user=www-data
+psql.pass=www-data
+
+# SMTP configuration
+smtpHost=localhost
+smtpPort=25
+
+# Other
+atlas.baseUrl=http://localhost:8080/atlas
```
Comment (fvanderbiest) [inline atlas/atlas.properties]: https://georchestra.mydomain.org/atlas to match what is done elsewhere in the current datadir

File: atlas/emails/finished.vm [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Hello,
```
Comment (fvanderbiest) [inline atlas/emails/finished.vm]: with a file with .vm extension ? template emails are using .tpl extension in extractorapp & ldapadmin

File: atlas/logback/logback.xml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+<configuration>
+
+    <appender name="standardOut" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <Pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</Pattern>
+        </encoder>
+    </appender>
+
+    <appender name="File" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <file>build/logs/logFile.log</file>
```
Comment (fvanderbiest) [inline atlas/logback/logback.xml]: relative path ?

Comment (fvanderbiest) [inline atlas/logback/logback.xml]: by default the datadir stores logs in /tmp eg with https://github.com/georchestra/datadir/blob/15.12/analytics/log4j/log4j.properties#L8

File: atlas/emails/finished.vm (hunk 2 above)
Comment (Vampouille) [inline atlas/emails/finished.vm]: @pmauduit does velocity care about extension ?

File: atlas/atlas.properties [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+atlas.emailFrom=noreply+atlas@georchestra.org
+atlas.emailSubject=[geOrchestra] Your Atlas request
+
+atlas.temporaryDirectory=/tmp/georchestra/atlas
```
Comment (fvanderbiest) [inline atlas/atlas.properties]: what if this dir does not exist ? Is it safer with /tmp ?

File: atlas/emails/finished.vm (hunk 2 above)
Comment (pmauduit) [inline atlas/emails/finished.vm]: Velocity Engine usually uses the `vm` extension, but I guess it should not complain for a .tmpl file. I'd prefer to keep this convention though.

File: atlas/logback/logback.xml (hunk 3 above)
Comment (Vampouille) [inline atlas/logback/logback.xml]: yes and in `build` directory which does not exists. I don't known where to put this file ...

Comment (Vampouille) [inline atlas/logback/logback.xml]: With 3e3b262, log are stored in `/tmp/atlas.log`

**Comment by fvanderbiest** [comment]
Status of this PR ?
To be merged ?

**Comment by Vampouille** [comment]
Yes, it's just new file and atlas is already merge on main repo

**Pull request merged**